.nuspec/Microsoft.Maui.Resizetizer.props

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <PropertyGroup>
+        <PrepareMsixPackageDependsOn>
```

Comment:
This is needed in a new props because the WinUI targets use this value and thus we only inject ourselves after the task dependency tree is built.

Reply:
If this pack changes back to a workload pack (right now is a library pack), you won't be able to have a `.props` file. Does it completely break if this property is set in the `.targets`?

Reply:
Unfortunately, yes. I might be able to try again and use a BeforeTargets and then have that depend on the teal task so it runs as a dependency only. Let me do that.